### PR TITLE
User story9

### PR DESCRIPTION
**Pull request by zachlp2016**
Updated to resolved error messages.

### CODE REVIEW

**Review by WHomer — APPROVED** [review]
Looks Gouda.

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -19,4 +19,15 @@
...
   end
+
+  describe 'class methods' do
+
+    it 'email_string' do
+      user_1 = User.create!(name: "default_user", role: 0, active: true, password_digest: "8320280282", address: "333", city: "Denver", state: "CO", zip: "80000", email: "default_user@gmail.com" )
+      user_2 = User.create!(name: "default_user1", role: 0, active: true, password_digest: "8320280282", address: "333", city: "Denver", state: "CO", zip: "80000", email: "default_user1@gmail.com" )
+      user_3 = User.create!(name: "default_user2", role: 0, active: true, password_digest: "8320280282", address: "333", city: "Denver", state: "CO", zip: "80000", email: "default_user2@gmail.com" )
+
+      expect(User.email_string).to eq(["default_user@gmail.com", "default_user1@gmail.com", "default_user2@gmail.com"])
```
Comment (BrennanAyers) [inline spec/models/user_spec.rb:30]: Cool! Yeah, this is much better.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -39,6 +42,12 @@ def update
 
   private
 
+  def email_confirmation
+    User.email_string.all? do |email|
```
Comment (BrennanAyers) [inline app/controllers/users_controller.rb:46]: We can refactor another time:

**Review by BrennanAyers — APPROVED** [review]
Swell!

**Pull request merged**